src/assignments/6-array.js

```diff
@@ -21,3 +26,9 @@ min=1
 
 */
 // start here writing the code for program2
+const numberArray = [1, 4, 2, 3]
+const min = Math.min(...numberArray)
+const max = Math.max(...numberArray)
+
+console.log(`Minimum number in the array is ${min}`)
+console.log(`Maximum number in the array is ${max}`)
```

Comment:
here, we have to write the program to get the min and the max of the array, you can try to run the for loop.
You can get the logic from here:
https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php